app/src/main/java/com/example/plantsapp/presentation/ui/utils/ImageLoadingExt.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.plantsapp.presentation.ui.utils
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.net.Uri
+import android.widget.ImageView
+import androidx.annotation.DrawableRes
+import com.bumptech.glide.Glide
+import com.bumptech.glide.RequestBuilder
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.example.plantsapp.R
+
+fun <T> ImageView.loadPicture(
+    context: Context,
+    picture: T,
+    @DrawableRes placeholder: Int = DEFAULT_PLACEHOLDER
+) {
+    Glide.with(context)
+        .load(picture)
+        .disableCache()
+        .centerCrop()
+        .placeholder(placeholder)
+        .into(this)
+}
+
+fun Uri?.getBitmap(
```

Comment:
https://stackoverflow.com/a/53882773/13101450

I googled for this solution on stack overflow , and somebody said to clear target after you get bitmap , I'm not is it needed when cache is disabled, but check this and maybe add

Reply:
Also maybe rename this from getBitmap to something like getBitmapWithPlaceholder